src/http/iot/http.cpp

```diff
@@ -84,7 +84,7 @@ class PlatformHTTP : public AbstractHTTP
             http.setTimeout(3000);
             http.begin(this->host_, this->port_, request);
 
-            http.addHeader("Content-Type", "application/x-www-form-urlencoded");
+            http.addHeader("Content-Type", "application/json");
```

Comment:
This is from https://github.com/ArkEcosystem/cpp-client/pull/21 and doesn't belong here.

Reply:
removed

https://github.com/ArkEcosystem/cpp-client/pull/22/commits/3b33849a86586994e5dca2bb32392871e99e9bc8